Source/Core/VideoBackends/D3D12/DX12Texture.cpp

```diff
@@ -0,0 +1,334 @@
+// Copyright 2009 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/Align.h"
+#include "Common/Assert.h"
```

Comment:
`<algorithm>`, `<cstddef>`, `<cstring>` and `Common/CommonTypes.h` needs to be included as well.